How to use it

KaiKangSDU opened the pull request:
Hi, thanks for your modifying code to suit python3. It really works. But I have a question. How do I use it ?How can I install it? Or I just add it to my code folder? Thanks in advance.

salaniz committed:
Trim project to only keep pycocoevalcap related files

salaniz committed:
Fix Python 3 syntax errors

salaniz committed:
Replace equivalent functions of Python 3

salaniz committed:
Fix Meteor score

salaniz committed:
Update README

flauted commented:
Consider using my fork from PR#32. The code is packaged so you can either clone it and set your PYTHONPATH appropriately, or you can pip install it from Github.

salaniz committed:
Add setup.py for easier installation

salaniz committed:
Update README and add pycocotools dependency to setup.py

salaniz committed:
Merge remote-tracking branch 'upstream/master'

salaniz committed:
Make SPICE python 3 compatible and add scipt to download stanford models

salaniz committed:
Fix missing import and bump version to 1.1

salaniz committed:
Fix Windows file path for extracting from ZipFile (resolves #7)

nicolas-lair committed:
Update bleu.py

Allow a parameter in compute score method to turn off verbose in Bleu_scorer compute_score method

salaniz committed:
Merge pull request #9 from nicolas-lair/patch-1

Allow verbose off for Bleu score

salaniz committed:
Add example script and update pip installation method

j-min committed:
Add verbosity control to PTBTokenizer

PTBTokenizer logs tokenization details by default (ex. `PTBTokenizer tokenized 2 tokens at 33.87 tokens per second`).
This becomes noisy when you have run tokenization iteratively.
I redirect stderr to `subprocess.DEVNULL` to suppress this.

j-min committed:
Enable PTBTokenizer stderr by default

j-min committed:
Fix bug: verbose -> self.verbose

salaniz committed:
Merge pull request #12 from j-min/master

Enable verbosity control of PTBTokenizer
